### PR TITLE
[media] Update uploaded form TestPlan

### LINKED ISSUE #6948: [media] Testing plan update for Upload form
**Describe the bug**
Media module Testing Plan states that site is a required field for the Upload form. However, site is no longer a field because it is inferred from the visit.

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #6948  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: modules/media/test/TestPlan.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
     - _Note: only a person with direct database access could revert the hide file action._
 
 **Test filters** [Automation Testing]
-  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: File Name, Language, Project, File Visibility, PSCID, Instrument, Uploaded By, Visit Label, Site and File type.
+  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: File Name, Language, Project, File Visibility, PSCID, Instrument, Uploaded By, Visit Label, and File type.
```
Comment (maltheism) [inline modules/media/test/TestPlan.md:85]: Hi @haoweiqiu, I think for filters `Site` exists.
media_table

**Review by maltheism — APPROVED** [review]

**Review by christinerogers — APPROVED** [review]
Hi Haowei - this looks good to me. 

In keeping with our conventions, could you add 1-2 lines at the top of your PR header so that it's clear

File: modules/media/test/TestPlan.md [hunk 2]
```diff
@@ -42,15 +42,15 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
 **Testing with write-permission** [Automation Testing]
   1. Add write permission
   2. Media module should now have **two** tabs (Browse) and (Upload)
-  3. Clicking on Upload tab should hide the data table and display a form with the following fields: PSCID, Visit Label, Instrument, For Site, Data of Administration, Comments, File to Upload
+  3. Clicking on Upload tab should hide the data table and display a form with the following fields: PSCID, Visit Label, Instrument, Data of Administration, Comments, File to Upload
```
Comment (HenriRabalais) [inline modules/media/test/TestPlan.md:45]: Data of Administration, or Date of Administration?

**Review by HenriRabalais — CHANGES_REQUESTED** [review]
Just a small correction, then looks good!

**Comment by christinerogers** [comment]
@haoweiqiu  - could you address henri's comments this week?

**Comment by christinerogers** [comment]
@driusan I think this could be final-reviewed for merge.

**Pull request merged**